dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/TaskNode.java

```diff
@@ -141,6 +142,13 @@
      */
     private int delayTime;
 
+    /**
+     * The timeout setting for the dependent node to wait for the start of pre-process.
+     */
+    @JsonDeserialize(using = JSONUtils.JsonDataDeserializer.class)
+    @JsonSerialize(using = JSONUtils.JsonDataSerializer.class)
+    private String waitStartTimeout;
```

Comment:
Should This feature be supported by front-end?

Reply:
There will be another person to complete the corresponding front-end feature.